Lil-Peep/src/styles/modules/featured-categories.scss

```diff
@@ -0,0 +1,206 @@
+@media only screen and (max-width: 768px) {
+  .categories_main {
+    border-color: blue;
+    border-style: solid;
+    border: 5px;
+    background-color: $color_bg;
+    border-bottom: 1px solid $color_bd;
+    font-family: "Oswald", sans-serif;
+    margin-top: 15px;
+
+    .banner_wooden_toys {
+      background-position: right;
+      background-repeat: no-repeat;
+      background-size: cover;
+      width: 100%;
+      text-decoration: none;
+
+      .wooden_toys_content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
+        display: flex;
+        height: 160px;
+        width: 100%;
+
+        .cont {
+          text-decoration: none;
+        }
+
+        .title {
+          color: white;
+          margin-bottom: 5px;
+          margin-left: 8px;
+        }
+      }
+    }
+
+    .banner_plush_toys {
+      background-position: 50% 30%;
+      background-repeat: no-repeat;
+      background-size: 100%;
+
+      .plush_toys_content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
+        display: flex;
+        height: 160px;
+        width: 100%;
+      }
+
+      .cont {
+        text-decoration: none;
+      }
+
+      .title {
+        color: white;
+        margin-bottom: 5px;
+        margin-left: 8px;
+        text-decoration: none;
+      }
+    }
+
+  }
+
+  .banner_outdoors {
+    background-position: 70% 30%;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+    .outdoors_content {
```

Comment:
Selector `outdoors_content` should be written in lowercase with hyphens

Reply:
Important! :point_up: